src/sentry/runner/commands/devserver.py

```diff
@@ -158,24 +159,17 @@ def devserver(
     parsed_url = urlparse(url_prefix)
     # Make sure we're trying to use a port that we can actually bind to
     needs_https = parsed_url.scheme == "https" and (parsed_url.port or 443) > 1024
-    has_https = False
-
-    if needs_https:
-        from subprocess import check_output
-
-        try:
-            check_output(["which", "https"])
-            has_https = True
-        except Exception:
-            has_https = False
-            from sentry.runner.initializer import show_big_error
-
-            show_big_error(
-                [
-                    "missing `https` on your `$PATH`, but https is needed",
-                    "`$ brew install mattrobenolt/stuff/https`",
-                ]
-            )
+    has_https = shutil.which("https") is not None
+
+    if needs_https and not has_https:
+        from sentry.runner.initializer import show_big_error
+
+        show_big_error(
+            [
+                "missing `https` on your `$PATH`, but https is needed",
```

Comment:
Nit/Optional: I'd probably add `, please run:` on the end of this error message.

Reply:
the only reason I found this is I'm pretty sure we don't want this message at all -- but was going to handle that in a followup.  we don't control that brew repository (an ex sentry employee's personal repo)

Reply:
Ah gotcha. I don't feel strongly but would probably remove this suggestion altogether given it's not a common fix for this.